website/content/en/v1.0/reference/settings.md

```diff
@@ -49,6 +49,10 @@ Karpenter uses [feature gates](https://kubernetes.io/docs/reference/command-line
 | Drift                   | true    | Beta  | v0.33.x |         |
 | SpotToSpotConsolidation | false   | Alpha  | v0.34.x |         |
 
+{{% alert title="Note" color="primary" %}}
+In v1 drift has been promoted to stable and the feature gate removed. Users can continue to control drift by using disruption budgets by reason.
```

Comment:
```suggestion
In v1, drift has been promoted to stable and the feature gate removed. Users can continue to control drift by using disruption budgets by reason.
```

Reply:
Do you think it makes sense to show an example of what this would look like?

Reply:
I provided an example just in v1.0 folder for now because I don't know if the YAML is properly display inside {{alert}} section. Please advice what should be used as a local tool to properly display changes.